app/src/main/java/org/greenstand/android/TreeTracker/database/TreeTrackerDAO.kt

```diff
@@ -34,9 +44,8 @@ interface TreeTrackerDAO {
     @Query("UPDATE planter_info SET uploaded = :isUploaded WHERE _id IN (:ids)")
     fun updatePlanterInfoUploadStatus(ids: List<Long>, isUploaded: Boolean)
 
-
     @Transaction
-    @Query("SELECT * FROM planter_check_in WHERE photo_url IS null AND planter_info_id = :planterInfoId")
+    @Query("SELECT * FROM planter_check_in WHERE photo_url IS null AND planter_info_id = :planterInfoId") // kt-lint-disable max-line-length
```

Comment:
For aesthetics and readability I let this Query violate the ktlint's line length. It will be odd to break the single line annotation to two lines.